guides/saas/getting-started.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Getting started with Coder's hosted beta
+description: Get started with a hosted Coder deployment.
+---
+
+To help you get started, we've put together this guide to help you understand
+how the Coder hosted beta works and how to access your deployment.
+
+## What is Coder hosted beta?
+
+The Coder hosted beta is a hybrid cloud offering of Coder's self-hosted
+platform. With this model, Coder hosts the control plane, and you host your
+compute and development workspaces. The benefits include:
+
+- Automatic upgrades
+- No installation or DNS configuration
+- Managed logging & monitoring
+
+Curious how this works? Here's a breakdown of the architecture:
```

Comment:
I suggest adding a CTA for folks to try it, as we do on out home page

```suggestion
If you interested in trying the limited beta, [contact us](https://coder.com/contact?note=I%20would%20like%20to%20try%20the%20hosted%20offer.%0A%0ANumber%20of%20developers%3A%0A%0AUse%20case%3A))

Curious how this works? Here's a breakdown of the architecture:
```

Reply:
JFYI: this probably isn't linted correctly. i did from the web ui

Reply:
Good call - will add.